docs/repro.8.txt

```diff
@@ -37,6 +37,9 @@ Options
         directory. This one is used to build the package instead of checking out
         a PKGBUILD from asp.
 
+ *-n*::
+        When enable,d `repro` will run makepkg with --nocheck.
```

Comment:
typo: enabled

Reply:
Thanks fixed locally. Will respin/submit one the NOCHECK is merged.